### ISSUE TITLE
Documentation on Contributing - Needs Updating

### ISSUE THREAD

**Issue by jvanderaa** [networktocode/circuit-maintenance-parser#209]
### Environment

- Python version:  3.x
- circuit_maintenance_parser version:  latest

### Expected Behavior

I expected that the invoke tasks would allow for the testing

### Observed Behavior
- Errors due to no Dockerfile

### Steps to Reproduce

1. clone repo
2. run `invoke build`
3. Get error that Dockerfile was not present

**Comment by chadell** [comment]
True, the library runs simple pytest testing without a container.

**Comment by csessh** [comment]
@jvanderaa 

I made a quick update with this PR: https://github.com/networktocode/circuit-maintenance-parser/pull/212

Does it look alright to you?

**Comment by slyngshede** [comment]
On macOS using Poetry to install will fail. I haven't debugged further, but building numpy fails, though I'm not sure what it is that Poetry is trying to install as it doesn't include a number of required packages like numpy. 

Installing packages with pip in a virtualenv work fine:
```
pip install pydantic chardet geopy tzwhere backoff icalendar netconan invoke toml click lxml pytest
pytest --log-cli-level=DEBUG --capture=tee-sys -k test_parsers
```

**Comment by chadell** [comment]
hi @slyngshede , which Poetry version are you using?

**Comment by slyngshede** [comment]
@chadell You can ignore my comment, it was related to running Python 3.11 and newer versions of NumPy.
